calculator/src/test/resources/application-testIT.properties

```diff
@@ -0,0 +1,8 @@
+database-backup-url=src/test/resources/backUpRecipeBaseTest.json
+product-name=product
```

Comment:
I agree when key logically group several levels of properties. In this case, we do not have separation of levels e.g. spring.application.name, because there are spring.profiles.active, but only the last part of the key which according to the convention describe the purpose of the property. Here one could be confused because product-name looks like property name of property product, but there is no property product, it is just a description of the field. Otherwise one could think there is a product defined for example in @ConfigurationProperties(prefix = "product"), which is not the case.

Reply:
We already have a set convention for the naming of the properties, as you can see in the application.properties file. product-name is a property, all properties are used to describe something, but the logic is extracted to a properties file due to maintainability.  
Your other point for the separation of levels is also not valid, because we need to keep our code as extensible and granular as possible. 
Please change the names of the attributes to follow the convention already used in the project.
I also noticed that in the application.properties you've added a field following the "-" convention e.g. database-backup-url - please change that too.

Reply:
Done